ArtOfIllusion/src/artofillusion/math/TriangleMath.java

```diff
@@ -0,0 +1,134 @@
+/* Copyright (C) 2016 by Petri Ihalainen
+
+   This program is free software; you can redistribute it and/or modify it under the
+   terms of the GNU General Public License as published by the Free Software
+   Foundation; either version 2 of the License, or (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+   PARTICULAR PURPOSE.  See the GNU General Public License for more details. */
+
+package artofillusion.math;
+
+import artofillusion.*;
+import artofillusion.ui.*;
+import artofillusion.math.*;
+
+/**
+ *  TriangleProps contains frequently used calculations about triangles
+ */
+public class TriangleMath
+{
+	/** Constructor for TriangleProps */
+	public void TriangleMath(){}
```

Comment:
This is not actually a constructor at all, it is a method named `TriangleMath()` with a `void` return type. Unless you are adding state (a data structure to hold a generic triangle, etc) you probably want a *private* constructor, so that no one can create a TriangleMath object. Like so:

    private TriangleMath() {}